routes/api/comments.js

```diff
@@ -34,7 +34,10 @@ router.post('/comments/:id/likes', restoreUser, asyncHandler(async(req, res) =>
 
 
 //Unlike a comment
-router.delete('/comments/likes/:id', asyncHandler(async(req, res) => {
+router.delete('/comments/likes/:id', restoreUser, asyncHandler(async(req, res) => {
+    const likedComments = await CommentLike.findAll();
+    const like = likedComments.some(likedComment => likedComment.id === parseInt(req.params.id));
+    if (like) {
```

Comment:
If like does not exist, we should send a `404` error.

Reply:
Send 404 error:
Line 31: if (!like) {res.status(404)};